sites/dentistryiq/server/templates/magazine/index.marko

```diff
@@ -18,9 +18,7 @@ $ const description = site.get('magazines.description');
         <div class="page-wrapper__header">
           <endeavor-breadcrumbs-magazine />
           <h1 class="page-wrapper__title">Magazines</h1>
-          <if(description)>
-            <p class="page-wrapper__description">${description}</p>
```

Comment:
Since the `${description}` is no longer used, you can safely remove the associated constant declarations on L1 and L2